app/templates/participate/login.hbs

```diff
@@ -24,7 +24,7 @@
     <div class="row">
       <div class="col-md-12">
         <h1>International Situations Project</h1>
-        {{isp-jam-auth login=(action 'authenticate')}}
+        {{isp-jam-auth invalidAuth=invalidAuth authenticating=authenticating login=(action 'authenticate')}}
```

Comment:
It's used in https://github.com/CenterForOpenScience/exp-addons/blob/develop/exp-models/addon/components/jam-auth/component.js#L17 and it's passed in so that the button remains disabled while the invalid auth modal is open (and then it will be re-enabled when the modal is dismissed)